pipenv/routines/update.py

```diff
@@ -42,6 +42,10 @@ def do_update(
         site_packages=site_packages,
         clear=clear,
     )
+    if packages is None:
```

Comment:
In my opinion, this would be more pythonic if we go like this :)

```python
packages = [p for p in (packages or []) if p]
editable = [p for p in (editable_packages or []) if p]
```

Reply:
Thanks!